.env

```
@@ -3,4 +3,5 @@ REACT_APP_REQUEST_BROKER_BASEURL=http://rac-vch.ad.rockarchive.org:8001/api
 REACT_APP_LOCALSTORAGE_KEY=DIMESMyList
 REACT_APP_MINIMAP_KEY=DIMESMinimapIntro
 REACT_APP_S3_BASEURL=https://iiif.rockarch.org
-REACT_APP_CAPTCHA_SITE_KEY=6LdQiSkTAAA
\ No newline at end of file
+REACT_APP_CAPTCHA_SITE_KEY=6LdQiSkTAAA
+REACT_APP_ENABLE_READING_ROOM_SELECT=false
```

```suggestion
REACT_APP_ENABLE_READING_ROOM_SELECT=true
```
Suggesting changing this to true to reduce confusion, since the string `false` will actually evaluate to a boolean `True` in this implementation.